src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp

```diff
@@ -196,6 +190,14 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
         bool group_events = (all_events.size() > 1);
         return aggregate_events(all_events, stream, group_events);
     }
+
+    void add_kernels(kernels_cache& kernels_cache) override {
```

Comment:
Lets modify the function name to avoid confusion, "add_kernel_codes", becuase "add_kernel" is being used to add kernel binary pointer

Reply:
Update it on master branch